src/promptflow/promptflow/batch/_base_executor_proxy.py

```diff
@@ -271,3 +408,16 @@ def _process_http_response(self, response: httpx.Response):
                     message_format=message_format, status_code=response.status_code, error=response.text
                 )
                 return ExceptionPresenter.create(unexpected_error).to_dict()
+
+    def _process_http_stream_response(self, response, status_code):
+        # if the status code is not 200, log the error
+        message_format = "Unexpected error when executing a line, status code: {status_code}, error: {error}"
+        bulk_logger.error(message_format.format(status_code=status_code, error=response))
+        # if response can be parsed as json, return the error dict
+        # otherwise, wrap the error in an UnexpectedError and return the error dict
+        try:
```

Comment:
What's the difference with `_process_http_response`?

Reply:
merged. we can't use reponse.text directly for streaming response